backend-rust/src/graphql_api/account_metrics.rs

```diff
@@ -90,12 +69,13 @@ impl QueryAccountMetrics {
         .await?;
 
         let x_time = rows.iter().map(|r| r.bucket_time).collect();
-        let y_last_cumulative_accounts_created =
-            rows.iter().map(|r| r.end_index.expect("coalesced")).collect();
-        let y_accounts_created = rows
-            .iter()
-            .map(|r| r.end_index.expect("coalesced") - r.start_index.expect("coalesced"))
-            .collect();
+        let y_last_cumulative_accounts_created: Vec<i64> =
+            rows.iter().map(|r| r.end_index).collect();
+        let y_accounts_created: Vec<i64> =
+            rows.iter().map(|r| r.end_index - r.start_index).collect();
+        let last_cumulative_accounts_created =
+            *y_last_cumulative_accounts_created.last().unwrap_or(&0);
```

Comment:
This default is a mistake

Reply:
I was wrong about this. It will fetch the latest even though it is from another bucket, and even if it is from outside the interval. Returning 0 is exactly correct.

Reply:
Maybe comment above to why this is correct, it does take some reasoning to arrive at